trafgen.c

```diff
@@ -1056,9 +1058,24 @@ int main(int argc, char **argv)
 			ctx.verbose = true;
 			break;
 		case 'P':
-			cpus_tmp = strtoul(optarg, NULL, 0);
-			if (cpus_tmp > 0 && cpus_tmp < ctx.cpus)
-				ctx.cpus = cpus_tmp;
+			if (slow) {
+				if (ctx.verbose)
+					printf("Option with slow mode was requested, "
+					       "skip cpu configuration\n");
```

Comment:
Please drop this message. There is no user-visible knob for slow mode (the smoke test mode will set it as well if a shaper is set) and it might not be determined at this point of processing options. Also, it will not be printed if `-P` is specified before `-s`.

Reply:
I see, thank you for clarification about slow mode. I've removed this message and similar issues.